src/guiguts/tools/pptxt.py

```diff
@@ -1271,32 +1205,32 @@ def unicode_numeric_character_check() -> None:
     )
 
     # Courtesy limit if user uploads fpgen source, etc.
```

Comment:
Double-checking that `fpgen` is what's meant here. Is there a more general description?